easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java

```diff
@@ -125,11 +125,13 @@ public static void requestPermissions(@NonNull final Object object, @NonNull Str
      *         request code to track this request, must be < 256.
      * @param perms
      *         a set of permissions to be requested.
+     * @return true if a permissions request was started, false otherwise.
      */
     @SuppressLint("NewApi")
-    public static void requestPermissions(@NonNull final Object object, @NonNull String rationale,
+    public static boolean requestPermissions(@NonNull final Object object, @NonNull String rationale,
```

Comment:
@samtstern, so is issue #62 no longer relevant? (`hasPermissions()` already checks for the API level)